src/ol/renderer/canvas/VectorTileLayer.js

```diff
@@ -173,29 +159,16 @@ class CanvasVectorTileLayerRenderer extends CanvasTileLayerRenderer {
     const projection = viewState.projection;
     const layer = this.getLayer();
     const tile = layer.getSource().getTile(z, x, y, pixelRatio, projection);
-    if (tile.getState() < TileState.LOADED) {
+    const viewHints = frameState.viewHints;
+    const hifi = !(
+      viewHints[ViewHint.ANIMATING] || viewHints[ViewHint.INTERACTING]
+    );
+    if (hifi || !tile.wantedResolution) {
       tile.wantedResolution = resolution;
```

Comment:
Do we not need to know `tile.wantedResolution` any more?

Reply:
We do need to know it, and we still set it in `getTile()`.